### DISCUSSION TITLE
Is it possible to call a HA automation using an Alexa voice command?

### DISCUSSION THREAD

**Discussion by dreed47** [alandtse/alexa_media_player#1799] [Q&A] [answered]
Is it possible to call a HA automation using an Alexa voice command for example "Alexa, turn off my outside lights" where "turn off my outside lights" is in fact a HA automation?

**Comment by broyuken** [comment, marked as answer]
You could make the alexa routine push a button in HA which then runs an automation. Thats what I do with my "Alexa Goodnight" routine. It pushes a HA button which is a trigger for my automation.
